lib/wallet/src/Cardano/Wallet/Read/Primitive/Tx/Features/Integrity.hs

```diff
@@ -0,0 +1,71 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE TypeFamilies #-}
+
+-- |
+-- Copyright: © 2020-2022 IOHK
+-- License: Apache-2.0
+--
+
+module Cardano.Wallet.Read.Primitive.Tx.Features.Integrity
+    ( integrity
+    , txIntegrityF
+    , txIntegrity
+    , txIntegrityCardanoApi
```

Comment:
ℹ️ I think it's a bit overkill to have `txIntegrityCardanoApi` specialized and exported here — as Cardano.Api is not a native inhabitant of the `Read` hierarchy, it would be favorable to define it locally or even inline it at the call site, which is a single unit test. But I don't see much harm in exporting either, especially as we want to gradually remove `Primitive` anyway, so I'm happy to approve as is.

Reply:
I moved the Api function out to call place. Unexported the specialized EraFun but commented for export in case of reuse.